Base.rte/Devices/Tools/GrappleGun/Grapple.lua

```diff
@@ -1,280 +1,440 @@
 function Create(self)
 
-	self.mapwrapx = SceneMan.SceneWrapsX;
-	self.CTimer = Timer();
-	self.MouseCTimer = Timer();
-	self.actionmode = 0;
+	self.mapWrapsX = SceneMan.SceneWrapsX;
```

Comment:
I'm just skimming this file cause I don't wanna deal with its 440+ lines at the moment.